### ISSUE TITLE
Offline edit not opening and app is crashing.

### ISSUE THREAD

**Issue by PrajwalM2212** [openfoodfacts/openfoodfacts-androidapp#972]
The app crashes when offline edit is chosen from navigation drawer.
```FATAL EXCEPTION: main
                                                                                     Process: openfoodfacts.github.scrachx.openfood, PID: 4379
                                                                                     android.view.InflateException: Binary XML file line #21: Binary XML file line #21: Error inflating class openfoodfacts.github.scrachx.openfood.utils.CustomButtonView
                                                                                     Caused by: android.view.InflateException: Binary XML file line #21: Error inflating class openfoodfacts.github.scrachx.openfood.utils.CustomButtonView
                                                                                     Caused by: java.lang.reflect.InvocationTargetException
                                                                                         at java.lang.reflect.Constructor.newInstance0(Native Method)
                                                                                         at java.lang.reflect.Constructor.newInstance(Constructor.java:334)
                                                                                         at android.view.LayoutInflater.createView(LayoutInflater.java:647)
                                                                                         at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:790)
                                                                                         at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:730)
                                                                                         at android.view.LayoutInflater.rInflate(LayoutInflater.java:863)
                                                                                         at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
                                                                                         at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
                                                                                         at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
                                                                                         at openfoodfacts.github.scrachx.openfood.fragments.BaseFragment.createView(BaseFragment.java:17)
                                                                                         at openfoodfacts.github.scrachx.openfood.fragments.OfflineEditFragment.onCreateView(OfflineEditFragment.java:55)
                                                                                         at android.support.v4.app.Fragment.performCreateView(Fragment.java:2261)
                                                                                         at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1419)
                                                                                         at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1750)
                                                                                         at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1819)
                                                                                         at android.support.v4.app.BackStackRecord.executeOps(BackStackRecord.java:797)
                                                                                         at android.support.v4.app.FragmentManagerImpl.executeOps(FragmentManager.java:2590)
                                                                                         at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2377)
                                                                                         at android.support.v4.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2332)
                                                                                         at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2239)
                                                                                         at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:700)
                                                                                         at android.os.Handler.handleCallback(Handler.java:789)
                                                                                         at android.os.Handler.dispatchMessage(Handler.java:98)
                                                                                         at android.os.Looper.loop(Looper.java:164)
                                                                                         at android.app.ActivityThread.main(ActivityThread.java:6541)
                                                                                         at java.lang.reflect.Method.invoke(Native Method)
                                                                                         at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
                                                                                         at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
                                                                                      Caused by: android.content.res.Resources$NotFoundException: Drawable openfoodfacts.github.scrachx.openfood:drawable/ic_done_all with resource ID #0x7f080076
                                                                                      Caused by: android.content.res.Resources$NotFoundException: File res/drawable/ic_done_all.xml from drawable resource ID #0x7f080076
                                                                                         at android.content.res.ResourcesImpl.loadDrawableForCookie(ResourcesImpl.java:768)
                                                                                         at android.content.res.ResourcesImpl.loadDrawable(ResourcesImpl.java:600)
                                                                                         at android.content.res.Resources.loadDrawable(Resources.java:885)
                                                                                         at android.content.res.TypedArray.getDrawableForDensity(TypedArray.java:953)
                                                                                         at android.content.res.TypedArray.getDrawable(TypedArray.java:928)
                                                                                         at openfoodfacts.github.scrachx.openfood.utils.CustomButtonView.initAttrs(CustomButtonView.java:47)
                                                                                         at openfoodfacts.github.scrachx.openfood.utils.CustomButtonView.<init>(CustomButtonView.java:28)
                                                                                         at java.lang.reflect.Constructor.newInstance0(Native Method)
                                                                                         at java.lang.reflect.Constructor.newInstance(Constructor.java:334)
                                                                                         at android.view.LayoutInflater.createView(LayoutInflater.java:647)
                                                                                         at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:790)
                                                                                         at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:730)
                                                                                         at android.view.LayoutInflater.rInflate(LayoutInflater.java:863)
                                                                                         at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
                                                                                         at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
02-27 17:09:21.684 4379-4379/openfoodfacts.github.scrachx.openfood E/AndroidRuntime:     at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
                                                                                         at openfoodfacts.github.scrachx.openfood.fragments.BaseFragment.createView(BaseFragment.java:17)
                                                                                         at openfoodfacts.github.scrachx.openfood.fragments.OfflineEditFragment.onCreateView(OfflineEditFragment.java:55)
                                                                                         at android.support.v4.app.Fragment.performCreateView(Fragment.java:2261)
                                                                                         at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1419)
                                                                                         at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1750)
                                                                                         at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1819)
                                                                                         at android.support.v4.app.BackStackRecord.executeOps(BackStackRecord.java:797)
                                                                                         at android.support.v4.app.FragmentManagerImpl.executeOps(FragmentManager.java:2590)
                                                                                         at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2377)
                                                                                         at android.support.v4.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2332)
                                                                                         at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2239)
                                                                                         at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:700)
                                                                                         at android.os.Handler.handleCallback(Handler.java:789)
                                                                                         at android.os.Handler.dispatchMessage(Handler.java:98)
                                                                                         at android.os.Looper.loop(Looper.java:164)
                                                                                         at android.app.ActivityThread.main(ActivityThread.java:6541)
                                                                                         at java.lang.reflect.Method.invoke(Native Method)
                                                                                         at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
                                                                                         at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
                                                                                      Caused by: java.lang.IllegalArgumentException: Float format error when parsing: - 6.34 1.41 1.41
                                                                                         at android.util.PathParser.nCreatePathDataFromString(Native Method)
                                                                                         at android.util.PathParser.-wrap1(Unknown Source:0)
                                                                                         at android.util.PathParser$PathData.<init>(PathParser.java:76)
                                                                                         at android.graphics.drawable.VectorDrawable$VFullPath.updateStateFromTypedArray(VectorDrawable.java:1924)
                                                                                         at android.graphics.drawable.VectorDrawable$VFullPath.inflate(VectorDrawable.java:1875)
                                                                                         at android.graphics.drawable.VectorDrawable.inflateChildElements(VectorDrawable.java:734)
                                                                                         at android.graphics.drawable.VectorDrawable.inflate(VectorDrawable.java:635)
                                                                                         at android.graphics.drawable.DrawableInflater.inflateFromXmlForDensity(DrawableInflater.java:142)
                                                                                         at android.graphics.drawable.Drawable.createFromXmlInnerForDensity(Drawable.java:1295)
                                                                                         at android.graphics.drawable.Drawable.createFromXmlForDensity(Drawable.java:1254)
                                                                                         at android.content.res.ResourcesImpl.loadDrawableForCookie(ResourcesImpl.java:758)
                                                                                         	... 34 more
```

**Comment by PrajwalM2212** [comment]
```Caused by: android.content.res.Resources$NotFoundException: File res/drawable/ic_done_all.xml from drawable resource ID #0x7f080076```
@shivanshsoni  Seems to be caused by your pull request #957 
Can you fix it ?

**Comment by Karljoones** [comment]
This is actually fixed in #961, just waiting for a merge
Poke @teolemon

**Comment by teolemon** [comment]
merged.
